README.md

```diff
@@ -113,7 +126,7 @@ end
 Your answer:
 ```text
 
-Replace this with your answer
+describe and context do the same thing. They visually lump tests together into blocks. It makes it visually easier to read for humans in English. For us it was easy to use "describe" for explaining a what a method does and "context" for things inside that method like the if/elsif part of the method.
 ```
```

Comment:
They're not the same. `describe` tells us what specific unit is being tested (e.g., the `Apartment` class, `Apartment.add_tenant`). `context` tells us about the conditions a certain unit is being tested under (e.g., `"when there is room (<= the number of beds)"`.